crates/teloxide-core/src/types/chat_boost_added.rs

```diff
@@ -0,0 +1,24 @@
+use serde::{Deserialize, Serialize};
+
+/// This object represents a service message about a user boosting a chat.
+#[serde_with::skip_serializing_none]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ChatBoostAdded {
+    /// Number of boosts added by the user
+    pub boost_count: u16,
```

Comment:
Needs some investigation, how many boosts can make a single user..:) Will `u8` not be suffice?